src/extensionWeb.ts

```diff
@@ -24,24 +27,20 @@ export async function activate(context: vscode.ExtensionContext) {
     )
 
     try {
-        setupGlobalStubs()
-
         // Setup the logger
         const toolkitOutputChannel = vscode.window.createOutputChannel('AWS Toolkit', { log: true })
         await activateLogger(context, toolkitOutputChannel)
 
+        setupGlobals()
+
         await initializeComputeRegion()
         initialize(context)
         initializeManifestPaths(context)
 
-        const awsContext = new DefaultAwsContext()
-        globals.awsContext = awsContext
+        await activateTelemetry(context, globals.awsContext, Settings.instance)
 
-        globals.sdkClientBuilder = new DefaultAWSClientBuilder(awsContext)
+        await initializeCredentials(context, globals.awsContext, globals.loginManager)
```

Comment:
It will help to move as much logic into `extensionShares.ts:activate()` as possible, soon, because if we wait much longer, then it will be less clear what steps are different accidentally vs intentionally.

Reply:
Good call. I will implement this in my next PR